app/controllers/event7/Event7MembersSummaryController.scala

```diff
@@ -36,7 +36,6 @@ import viewmodels.{Message, SummaryListRowWithThreeValues}
 import views.html.event7.{Event7MembersSummaryView, Event7MembersSummaryViewWithPagination}
 
 import javax.inject.Inject
-//TODO refactor this to use reusable membersSummary again, removing duplication of code- if statement in the view
```

Comment:
TODO removed without change in file - has a ticket been made to track this?

Reply:
Spoke to Emma who put this in. We agreed that it can be removed now.